File: PYTHON/Pertemuan 01/SegitigaLatihan.py
import sys
sys.stdout.reconfigure(encoding='utf-8')
class SegitigaLatihan:
    def __init__(self, alas: float, tinggi: float, sisi_a: float, sisi_b: float, sisi_c: float):
        self.alas = alas
        self.tinggi = tinggi
        self.sisi_a = sisi_a
        self.sisi_b = sisi_b
        self.sisi_c = sisi_c

    def hitung_luas(self) -> float:
        return 0.5 * self.alas * self.tinggi

    def hitung_keliling(self) -> float:
        return self.sisi_a + self.sisi_b + self.sisi_c

    def cetak(self):
        print(f"  Segitiga (a={self.alas:.1f}, t={self.tinggi:.1f}): Luas={self.hitung_luas():.2f}, Keliling={self.hitung_keliling():.2f}")

File: JAVA/Pertemuan 01/src/id/ac/uph/pbo/pertemuan01/SegitigaLatihan.java
package id.ac.uph.pbo.pertemuan01;

public class SegitigaLatihan {
    private double alas, tinggi, sisiA, sisiB, sisiC;
    public SegitigaLatihan(double alas, double tinggi, double sisiA, double sisiB, double sisiC) {
        this.alas   = alas;   this.tinggi = tinggi;
        this.sisiA  = sisiA;  this.sisiB  = sisiB; this.sisiC = sisiC;
    }
    public double hitungLuas()     { return 0.5 * alas * tinggi; }
    public double hitungKeliling() { return sisiA + sisiB + sisiC; }
    public void cetak() {
        System.out.printf("  Segitiga (a=%.1f, t=%.1f): Luas=%.2f, Keliling=%.2f%n",
                alas, tinggi, hitungLuas(), hitungKeliling());
    }
}
